.gitignore

```diff
@@ -22,3 +22,7 @@ erl_crash.dump
 
 # Ignore package tarball (built via "mix hex.build").
 shopify_api-*.tar
+
+# Dialyzer will create PLT files here 
+/priv/plts/*.plt
+/priv/plts/*.plt.hash
```

Comment:
could we just ignore all of /priv/plts ?

Reply:
I think we may want to add the folder /priv/plts/ and just not the files. That being said, I don't think that is what I have done.